.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and publish to ghcr.io
+
+on:
+  push:
+    branches: [ master ]
+  release:
```

Comment:
These two lines will trigger the workflow on release events as well. In this case $GITHUB_REF is set to the release's tag, and tags the docker image accordingly.

Reply:
@nawar-hamo-sonarsource : 👋